integration/v3_watch_test.go

```diff
@@ -1079,3 +1080,48 @@ func TestV3WatchWithFilter(t *testing.T) {
 		t.Fatal("failed to receive delete event")
 	}
 }
+
+func TestV3WatchWithPrevKV(t *testing.T) {
+	cluster := NewClusterV3(t, &ClusterConfig{Size: 1})
+	defer cluster.Terminate(t)
+	etcdcli := cluster.RandClient()
+
+	tests := []struct {
+		key  string
+		vals []string
+	}{{
+		key:  "foo",
+		vals: []string{"bar1", "bar2"},
+	}, {
+		key:  "/abc",
+		vals: []string{"first", "second"},
+	}}
+	for i, tt := range tests {
+		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+
+		_, err := etcdcli.Put(ctx, tt.key, tt.vals[0])
+		if err != nil {
+			t.Fatalf("fail to Put: %v", err)
+		}
+
+		wch := etcdcli.Watch(ctx, tt.key, clientv3.WithPrefix(), clientv3.WithPrevKV())
```

Comment:
`clientv3/integration` has native client tests.

`integration` does not depend on clientv3 code, As @heyitsanthony mentioned, for this test, we just want to test the correctness of server side and grpc calls?